implementation/workflows/send-dependency-update.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow triggers on a published release event.
+# Requires the following secrets:
+# - LANGUAGE_FAMILY_REPO -> the full name of the repo to send the dispatch to (eg. paketo-buildpacks/nodejs)
```

Comment:
Potentially this could be a list and the dispatch action calls multiple dispatchers